recipes/cdsdashboards/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "cdsdashboards" %}
+{% set version = "0.0.19" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ccc6eebb9e5471e6dcef8bc7d028561a8c8649b5d7b8dfcb3472ba3796fe7867
+
+build:
+  noarch: python
+  number: {{ build_number }}
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      number: {{ build_number }}
+      noarch: python
+    script: build_base.sh
+    requirements:
+      host:
+        - python >=3.6.0,<4.0.0
+        - pip
+      run:
+        - python >=3.6.0,<4.0.0
+        - tornado >=5.1
+        - traitlets
+        - jupyterhub >=1.0.0
+        - alembic
+        - pluggy
+    test:
+      imports:
+        - cdsdashboards
+  - name: {{ name|lower }}-singleuser
+    build:
+      number: {{ build_number }}
+      noarch: generic
```

Comment:
Is this noarch generic, not python? Just making sure!

Reply:
Sorry for the late answer. Yes it is generic as per a suggestion of @xhochy in [another recipe](https://github.com/conda-forge/staged-recipes/pull/11280/files/72f4a91638eb8846892cbc9bfe937e2b227768ae#diff-42f331d1ac0f9b69e3596208031be741)